code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -97,8 +98,8 @@
 		stance = HOSTILE_STANCE_ATTACK
 		if(isliving(target))
 			var/mob/living/L = target
-			if(L.bodytemperature > 261)
-				L.bodytemperature = 261
+			if(L.bodytemperature > 200)
+				L.bodytemperature = 200
```

Comment:
I know /tg/ made this change to offset a buff they made to body temperature stabilization, have we done something similar as well?